Imported css3-counter-styles-138.html is a constant failure

vitorroriz opened the pull request:
#### bb1f39b5a78bfb19c314a386f3ee5f1e487b89e4
<pre>
Imported css3-counter-styles-138.html is a constant failure
<a href="https://bugs.webkit.org/show_bug.cgi?id=245592">https://bugs.webkit.org/show_bug.cgi?id=245592</a>
rdar://100333890

Reviewed by Tim Nguyen.

Allowing some tolerance for test.

* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/mongolian/css3-counter-styles-138.html:
* LayoutTests/platform/ios-wk2/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/256446@main">https://commits.webkit.org/256446@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/909790f14af3c86f422541ddfc62d1854f39382a

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/95790 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/5042 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/28832 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/105368 "Built successfully") | [✅ 🛠 🧪 win](https://ews-build.webkit.org/#/builders/10/builds/165677 "Built successfully and passed tests") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/5127 "Built successfully") | [✅ 🛠 mac-debug](https://ews-build.webkit.org/#/builders/71/builds/33802 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/88174 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/101198 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/101450 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/78/builds/3775 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/82401 "Built successfully") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/30828 "Passed tests") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/9/builds/85634 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/87541 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/73659 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/66/builds/39533 "Built successfully") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/73/builds/19078 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/67/builds/37224 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/20401 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/4456 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/63/builds/41284 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/60/builds/43036 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/65/builds/43330 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/39653 "Passed tests") | | 
<!--EWS-Status-Bubble-End-->

nt1m commented on LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/mongolian/css3-counter-styles-138.html:
```diff
@@ -7,6 +7,7 @@
 <link rel='help' href='http://www.w3.org/TR/css-counter-styles-3/#simple-numeric'>
 <link rel='match' href='css3-counter-styles-138-ref.html'>
 <meta name="assert" content="list-style-type: mongolian produces numbers after 9 per the spec.">
+<meta name=fuzzy content="maxDifference=0-255;totalPixels=10-240">
```
The ranges should always start from 0, otherwise this means the test _must_ have at least 10px difference (which I don't think was your intent).

vitorroriz replied:
I thought to give it a 10 range such that we would be aware if tolerance is no longer needed. It is probably not a good practice, right?

nt1m replied:
The problem is that WPT are shared with other browsers, so this will make tests fail in other browsers with no fuzziness.

vitorroriz replied:
Fixed.

nt1m reviewed (approved):
I checked the image diff, and this looks reasonable.

Can you export this to WPT using the `export-w3c-test-changes` script? otherwise, this will get lost at the next import.

vitorroriz committed:
Imported css3-counter-styles-138.html is a constant failure
https://bugs.webkit.org/show_bug.cgi?id=245592
rdar://100333890

Reviewed by Tim Nguyen.

Allowing some tolerance for test.

* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/mongolian/css3-counter-styles-138.html:
* LayoutTests/platform/ios-wk2/TestExpectations:

Canonical link: https://commits.webkit.org/256446@main
